docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3"
+services:
+  #PHP Service
+  app:
```

Comment:
this is the block that gets all PHP libraries and dependencies and installs them into the virtual machine (docker image). Note, there will be a dedicated docker image (virtual machine) just to run PHP. Same will be true to the DB and Nginx